zipline/data/us_equity_minutes.py

```diff
@@ -0,0 +1,316 @@
+# Copyright 2015 Quantopian, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from abc import (
+    ABCMeta,
+    abstractmethod,
+)
+import bcolz
+import json
+import os
+from bcolz import ctable
+from datetime import datetime
+import numpy as np
+from numpy import float64
+from os.path import join
+import pandas as pd
+from pandas import read_csv
+from six import with_metaclass
+
+from zipline.finance.trading import TradingEnvironment
+from zipline.utils import tradingcalendar
+
+MINUTES_PER_DAY = 390
+
+_writer_env = TradingEnvironment()
```

Comment:
This should probably be a parameter to instances of the writer, rather than a global constant.  @jfkirk recently did a lot of work to remove global TradingEnvironment instances because of all the headaches they caused.